From 07a76d34a082848532326ef83ab7dafdb4e68b42 Mon Sep 17 00:00:00 2001
From: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Date: Thu, 15 Jun 2023 12:12:30 +0200
Subject: [PATCH 0379/1141] dt-bindings: bus: fix STM32 System Bus

Fix 'make dt_binding_check' that shows some errors due to missing headers.

Error: Documentation/devicetree/bindings/bus/
st,sys-bus.example.dts:41.29-30 syntax error
FATAL ERROR: Unable to parse input tree

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Change-Id: I8c01984967b3c6cabbaa811c6cbc67b648b88f6b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/312993
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 Documentation/devicetree/bindings/bus/st,sys-bus.yaml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/devicetree/bindings/bus/st,sys-bus.yaml b/Documentation/devicetree/bindings/bus/st,sys-bus.yaml
index ef876ebc8370..caa0d973391b 100644
--- a/Documentation/devicetree/bindings/bus/st,sys-bus.yaml
+++ b/Documentation/devicetree/bindings/bus/st,sys-bus.yaml
@@ -70,6 +70,11 @@ examples:
     // - same goes for rng1.
     // Access rights are verified before creating devices.
 
+    #include <dt-bindings/interrupt-controller/arm-gic.h>
+    #include <dt-bindings/clock/stm32mp1-clks.h>
+    #include <dt-bindings/reset/stm32mp1-resets.h>
+    #include <dt-bindings/bus/stm32mp15_sys_bus.h>
+
     etzpc: etzpc@0x5C007000 {
         compatible = "st,stm32mp15-sys-bus";
         reg = <0x5c007000 0x400>;
-- 
2.39.2

